src/crimson/os/seastore/btree/fixed_kv_btree.h

```diff
@@ -588,6 +588,13 @@ class FixedKVBtree {
                 : true);
             }
           }
+        } else if (child == RESERVATION_PTR) {
+          if constexpr(
+            !std::is_base_of_v<typename internal_node_t::base_t,
+                               child_node_t>) {
+            assert(i->get_val().pladdr.is_paddr());
+            assert(i->get_val().pladdr.get_paddr() == P_ADDR_ZERO);
+          }
```

Comment:
IIUC the `else` case means that `node` is leaf, and I suppose a similar check to P_ADDR_ZERO can also be applied there ?

Reply:
Um, it seems that there should be no `else` case, will try to assert it.